tests/integration/dhcp_test.py

```diff
@@ -89,10 +127,25 @@ def test_ipv4_dhcp(dhcp_env):
     dhcp_cli_desired_state['state'] = 'up'
     dhcp_cli_desired_state['ipv4']['enabled'] = True
     dhcp_cli_desired_state['ipv4']['dhcp'] = True
+    dhcp_cli_desired_state['ipv4']['auto-routes'] = True
+    dhcp_cli_desired_state['ipv4']['auto-dns'] = True
+    dhcp_cli_desired_state['ipv4']['auto-gateway'] = True
     dhcp_cli_desired_state['ipv6']['enabled'] = True
 
     netapplier.apply(desired_state)
     assertlib.assert_state(desired_state)
+    time.sleep(5)  # wait dhcp
```

Comment:
I do not understand this one. Isn't nmstate blocking until the profile is activated? Isn't activation also imply a response from the server with an IP assigned?

Reply:
The libnm return before IPv4 gateway was applied. No idea the root cause yet.

Reply:
Could you please update the comment to say something like `# wait to get resolv.conf updated` or similar.

Reply:
Done.